apps/rig_api/lib/rig_api/router.ex

```diff
@@ -32,4 +32,11 @@ defmodule RigApi.Router do
       delete("/:id", ApisController, :deactivate_api)
     end
   end
+
+  scope "/health", RigApi do
+    pipe_through(:api)
+    scope "/" do
```

Comment:
Would it also work without that inner `scope`? For example, see line 11.

Reply:
fixed - see latest commit